### ISSUE TITLE
Backport: Switch Canal log verbosity to warning

### ISSUE THREAD

**Issue by alena1108** [rancher/rancher#19988]
Backport https://github.com/rancher/rancher/issues/19472

**Comment by alena1108** [comment]
will be available in rc6

**Comment by jiaqiluo** [comment]
The bug is reproduced on Rancher: v2.2.2
- Canal
```
> kubectl -n kube-system get pods -l k8s-app=canal -o go-template='{{range .items}}{{range .spec.containers}}{{ if eq .name "calico-node"}}{{"===> Pod: "}}{{.name}}{{"\n"}}{{range .env}}{{.name}}{{"="}}{{if .value}}{{.value}}{{end}}{{if .valueFrom}}{{.valueFrom}}{{end}}{{"\n"}}{{end}}{{end}}{{end}}{{end}}' | egrep "Pod|FELIX_LOGFILEPATH|FELIX_LOGSEVERITYSYS|FELIX_LOGSEVERITYSCREEN"
===> Pod: calico-node
FELIX_LOGSEVERITYSCREEN=info
> kubectl -n kube-system get configmap canal-config -o go-template='{{.data.cni_network_config}}' | grep log_level
      "log_level": "info",
> for pod in $(kubectl -n kube-system get pods -l k8s-app=canal -o custom-columns=NAME:.metadata.name --no-headers); do echo "Log count for pod $pod and container calico-node:"; kubectl -n kube-system logs $pod -c calico-node | wc -l; done
Log count for pod canal-nhr8h and container calico-node:
1037
> 
```
- Calico
```
> kubectl -n kube-system get pods -l k8s-app=calico-node -o go-template='{{range .items}}{{range .spec.containers}}{{"===> Pod: "}}{{.name}}{{"\n"}}{{range .env}}{{.name}}{{"="}}{{if .value}}{{.value}}{{end}}{{if .valueFrom}}{{.valueFrom}}{{end}}{{"\n"}}{{end}}{{end}}{{end}}' | egrep "Pod|FELIX_LOGFILEPATH|FELIX_LOGSEVERITYSYS|FELIX_LOGSEVERITYSCREEN"
===> Pod: calico-node
FELIX_LOGSEVERITYSCREEN=info
> kubectl -n kube-system get configmap calico-config -o go-template='{{.data.cni_network_config}}' | grep log_level
      "log_level": "info",
> for pod in $(kubectl -n kube-system get pods -l k8s-app=calico-node -o custom-columns=NAME:.metadata.name --no-headers); do echo "Log count for pod $pod and container calico-node:"; kubectl -n kube-system logs $pod -c calico-node | wc -l; done
Log count for pod calico-node-2zfnn and container calico-node:
1316
> 
```


The bug fix is validated on Rancher: v2.2.3-rc6
- Canal
```
> kubectl -n kube-system get pods -l k8s-app=canal -o go-template='{{range .items}}{{range .spec.containers}}{{ if eq .name "calico-node"}}{{"===> Pod: "}}{{.name}}{{"\n"}}{{range .env}}{{.name}}{{"="}}{{if .value}}{{.value}}{{end}}{{if .valueFrom}}{{.valueFrom}}{{end}}{{"\n"}}{{end}}{{end}}{{end}}{{end}}' | egrep "Pod|FELIX_LOGFILEPATH|FELIX_LOGSEVERITYSYS|FELIX_LOGSEVERITYSCREEN"
===> Pod: calico-node
FELIX_LOGFILEPATH=none
FELIX_LOGSEVERITYSYS=
FELIX_LOGSEVERITYSCREEN=Warning
> 
> kubectl -n kube-system get configmap canal-config -o go-template='{{.data.cni_network_config}}' | grep log_level
      "log_level": "WARNING",
> for pod in $(kubectl -n kube-system get pods -l k8s-app=canal -o custom-columns=NAME:.metadata.name --no-headers); do echo "Log count for pod $pod and container calico-node:"; kubectl -n kube-system logs $pod -c calico-node | wc -l; done
Log count for pod canal-qxd4p and container calico-node:
20
> 
```

- Calico
```
>  kubectl -n kube-system get pods -l k8s-app=calico-node -o go-template='{{range .items}}{{range .spec.containers}}{{"===> Pod: "}}{{.name}}{{"\n"}}{{range .env}}{{.name}}{{"="}}{{if .value}}{{.value}}{{end}}{{if .valueFrom}}{{.valueFrom}}{{end}}{{"\n"}}{{end}}{{end}}{{end}}' | egrep "Pod|FELIX_LOGFILEPATH|FELIX_LOGSEVERITYSYS|FELIX_LOGSEVERITYSCREEN"
===> Pod: calico-node
FELIX_LOGFILEPATH=none
FELIX_LOGSEVERITYSYS=
FELIX_LOGSEVERITYSCREEN=Warning
> kubectl -n kube-system get configmap calico-config -o go-template='{{.data.cni_network_config}}' | grep log_level
      "log_level": "WARNING",
> for pod in $(kubectl -n kube-system get pods -l k8s-app=calico-node -o custom-columns=NAME:.metadata.name --no-headers); do echo "Log count for pod $pod and container calico-node:"; kubectl -n kube-system logs $pod -c calico-node | wc -l; done
Log count for pod calico-node-gkmfj and container calico-node:
90
> 

```
